pymc_experimental/tests/test_pathfinder.py

```diff
@@ -21,6 +22,7 @@
 import pymc_experimental as pmx
 
 
+@pytest.mark.skipif(platform.machine() != "x86_64", reason="Test requires x86_64 architecture")
```

Comment:
Why was this needed? If there's a github issue can you add as inline comment?

Reply:
Can you share the error message?

Reply:
Then I think we should remove the skip. The AVX instructions are a bit more CPU specific than just "x86_64"

Reply:
https://en.wikipedia.org/wiki/Advanced_Vector_Extensions
AVX were used only in x86, whether by intel or amd ( at least that's what I keep finding)

Reply:
Sorry I read the other way around, that you had a x86 without AVX. 

But then are you saying JAX cannot run on M1 Macs at all, or just your installation?